tests/src/com/unciv/uniques/CountableTests.kt

```diff
@@ -190,6 +190,38 @@ class CountableTests {
         assertEquals(10.5f, civCulture, 0.005f)
     }
 
+    @Test
+    fun testPoliciesCountables() {
+        setupModdedGame()
+        civ.policies.run {
+            for (name in listOf(
+                "Tradition", "Aristocracy", "Legalism", "Oligarchy", "Landed Elite", "Monarchy",
+                "Liberty", "Citizenship", "Honor", "Piety"
+            )) {
+                freePolicies++
+                val policy = getPolicyByName(name)
+                adopt(policy)
+            }
+            // Don't use a fake Policy without a branch, the policyFilter would stumble over a lateinit.
+            val taggedPolicyBranch = game.createPolicyBranch("Some marker")
+            freePolicies++
+            adopt(taggedPolicyBranch) // Will be completed as it has no member policies
+        }
+        val tests = listOf(
+            "Completed Policy branches" to 2,
+            "Adopted [Tradition Complete] Policies" to 1,
+            "Adopted [[Tradition] branch] Policies" to 7,
+            "Adopted [Liberty Complete] Policies" to 0,
+            "Adopted [[Liberty] branch] Policies" to 2,
+            "Adopted [[all] branch] Policies" to 5,
```

Comment:
How can All be lower than just Tradition?!

Reply:
This is still a blocker for me, something here is either broken or unintuitive enough that the interface needs changing

Reply:
We should fix it to return the number of policies that fit the filter, not the number of branches, if that's what it's doing